nss-cohort-36/bangazon-api-illin-illusionists-api
# Docs/readme

# Description

README and fixtures setup

## Type of change

## Testing Instructions
```
git fetch --all
git checkout docs/readme
rm -rf db.sqlite3
python manage.py makemigrations
python manage.py migrate
python manage.py loaddata bangazonAPI/fixtures/bangazon.json
```

1. Make sure the fixtures above all load at once with no errors
2. Check out the README file for the updated instructions for all of those users that are going to download and use our app (hah)
3. Check out your database in DBeaver/Tableplus and query the auth tables as well as bangazonAPI ables to see the new data that was loaded. Everything should also match up for the customers/users between django/bangazon (djangozon??!)


## Checklist:
